doc/lsst.obs.decam/using-obs-decam.rst

```diff
@@ -0,0 +1,79 @@
+
+This example assumes the user has setup the `~lsst.cp.pipe`_ module, which is needed for constructing calibration products from raws.
+
+.. prompt:: bash
+
+   butler ingest-raws . /path/to/raw/science/*.fits.fz --transfer link
+   butler ingest-raws . /path/to/raw/calibs/*.fits.fz --transfer link
+   pipetask run -b . -i DECam/raw/all -o DECam/raw/crosstalk-sources -p $CP_PIPE_DIR/pipelines/DarkEnergyCamera/RunIsrForCrosstalkSources.yaml
+
+Build nightly (or similar) bias and flat frames and certify them into a calib collection.
+
+This example assumes the user has a single night of observations with bias frames numbered 1-6 and flat frames numbered 7-12, and wants to create nightly calib products valid for 24 hours. Note the bias building pipeline is camera-agnostic, but the flat-building pipeline has a prerequisite DECam-specific step which correctly handles inter-chip crosstalk.
+
+.. prompt:: bash
+
+   pipetask run -d "exposure IN (1, 2, 3, 4, 5, 6)" -b . -i DECam/raw/all,DECam/calib -o u/username/bias-construction-night1 -p $CP_PIPE_DIR/pipelines/cpBias.yaml --register-dataset-types
+   butler certify-calibrations . u/username/bias-construction-night1 DECam/calib/run1 bias --begin-date 2021-01-01T00:00:00 --end-date 2021-01-01T23:59:59
+   pipetask run -d "exposure IN (7, 8, 9, 10, 11, 12)" -b . -i DECam/raw/all,DECam/raw/crosstalk-sources,DECam/calib -o u/username/flat-construction-night1 -p $CP_PIPE_DIR/pipelines/DarkEnergyCamera/cpFlat.yaml --register-dataset-types
+   butler certify-calibrations . u/username/flat-construction-night1 DECam/calib/run1 flat --begin-date 2021-01-01T00:00:00 --end-date 2021-01-01T23:59:59
+
+Now you can proceed with running ISR and other "processCcd" tasks via a Gen3 pipeline. Some useful pipelines can be found in the `pipelines/DarkEnergyCamera` directory of the `~lsst.ap.pipe`_ package.
+
+
+.. _lsst.obs.decam.gen2:
+
+############################
+The deprecated Gen2 workflow
+############################
+
+Create a Gen2 data repository directory:
+
+.. prompt:: bash
+
+   mkdir /path/to/repo
+   echo lsst.obs.decam.DecamMapper > /path/to/repo/_mapper
+
+Ensure you have obs_decam setup and built as well as obs_decam_data.
+
+Import raw and calibration data into the data repository, for example:
+
+.. prompt:: bash
+
+    ingestCuratedCalibs.py /path/to/repo --calib /path/to/calib/repo $OBS_DECAM_DATA_DIR/decam
+    ingestImages.py /path/to/repo --filetype raw /path/to/raw/*.fits.fz
+    ingestCalibs.py /path/to/repo/  --calib /path/to/calib/repo/ /path/to/ias-and-flat-files/*fits --validity 999
+
+By default, ingesting calibration data only creates a repository database.
+When ingesting biases and flats, if you would like to also link these files (in the same way as images are ingested), use `--mode=link`.
+
+Process data, noting you will want to turn astrometry and photometric calibration on if you have appropriate reference catalogs available:
```

Comment:
The "noting" about astrometry/photometry makes me wonder why this example has them disabled. Is it because the example doesn't have refcat data? You could like to Paul's hosting of some refcat files as linked here: https://community.lsst.org/t/gaia-dr2-reference-catalog-in-lsst-format/3901/6

Reply:
Again, not touching gen2 shenanigans, I'm quite sure this was written like this because community users were sufficiently overwhelmed with the gen2 ingest process and getting processCcd to run _at all_ was an achievement...